<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadarBeamRotateDlg</class>
 <widget class="QDialog" name="RadarBeamRotateDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RadarBeamRotateDlg</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>旋转Z</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>平移Y</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>旋转Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>平移Z</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>平移X</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>旋转X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_OffsetX"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_OffsetY"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>缩放倍数</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_OffsetZ"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_RotateY"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_RotateZ"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_Scale"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_RotateX"/>
     </item>
     <item row="7" column="1">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
